frontends/web/src/components/status/status.tsx

```diff
@@ -15,98 +15,71 @@
  * limitations under the License.
  */
 
-import { Component } from 'react';
-import { apiGet, apiPost } from '../../utils/request';
+import { ReactNode, useCallback, useEffect, useState } from 'react';
 import { CloseXWhite } from '../icon';
 import style from './status.module.css';
+import { getConfig } from '../../api/backend';
+import { setConfig } from '../../api/backend';
+
 
-interface State {
-    show: boolean;
-}
 
-interface StatusProps {
+type TPRops = {
     hidden?: boolean;
     type?: 'success' | 'warning' | 'info';
     // used as keyName in the config if dismissing the status should be persisted, so it is not
     // shown again. Use an empty string if it should be dismissable without storing it in the
     // config, so the status will be shown again the next time.
     dismissable?: string;
-    className?: string;
+  className?: string;
+  children: ReactNode;
 }
 
-type Props = StatusProps;
-
-export default class Status extends Component<Props, State> {
-  public readonly state: State = {
-    show: true,
-  };
-
-  public componentDidMount() {
-    this.checkConfig();
-  }
+const Status = ({ hidden, type = 'warning', dismissable, className, children }: TPRops) => {
+  const [show, setShow] = useState(true);
 
-  public componentDidUpdate(prevProps: Props) {
-    if (this.props.dismissable !== prevProps.dismissable) {
-      this.checkConfig();
+  const checkConfig = useCallback(async() => {
+    if (dismissable) {
+      const config = await getConfig();
+      setShow(!config ? true : !config[dismissable]);
     }
-  }
+  }, [dismissable]);
```

Comment:
could you change dismissable -> dismissible :) ?

code looks good to me, but I wonder why the old implementation had the componentDidUpdate. `dismissible` should not change during component livecycle, I think doing on mount would be enough. but maybe not in this commit, wdyt?

https://github.com/digitalbitbox/bitbox-wallet-app/search?q=dismissable&type=code
componentDidUpdate

Reply:
>  I think doing on mount would be enough. but maybe not in this commit, wdyt?


Yeah, I'd say so. I haven't exhaustively tested it yet though. 

And sure, I'll make the change for this in a new commit. :)